### ISSUE TITLE
Protelis line information missing again

### ISSUE THREAD

**Issue by jakebeal** [Protelis/Protelis#257]
Using Protelis 12, I'm getting missing line information in exceptions from Protelis again:

````
14:08:44.498 [DCOP-nodeB0] ERROR com.bbn.map.dcop.DCOPService - Got error computing DCOP plan. Skipping this round and will try again next round
org.protelis.lang.interpreter.util.ProtelisRuntimeException: getDcopInfoProvider[] does not exist in class com.bbn.map.dcop.CdiffPlusExecutionContext.
You tried to invoke it with arguments []
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:132) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:300) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:295) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:77) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:300) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.lambda$projectAndEval$2(AbstractAnnotatedTree.java:256) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree$$Lambda$3.accept(AbstractAnnotatedTree.java) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.forEachWithIndex(AbstractAnnotatedTree.java:164) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.projectAndEval(AbstractAnnotatedTree.java:255) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AssignmentOp.evaluate(AssignmentOp.java:45) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All.lambda$evaluate$0(All.java:52) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All$$Lambda$1.accept(All.java) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.forEachWithIndex(AbstractAnnotatedTree.java:164) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All.evaluate(All.java:50) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.vm.impl.SimpleProgramImpl.compute(SimpleProgramImpl.java:76) ~[protelis-interpreter-12.1.0.jar:?]
````

**Comment by DanySK** [comment]
The stacktrace is cut, I can't see the the ultimate cause. Can you post the complete stacktrace?
My feeling is that something at the application level is intercepting the exception and changing the printed stacktrace.

**Comment by jakebeal** [comment]
Here's the complete stack trace, which is actually two stack trace nodeB0 and noteA0 joined together (I believe serially):

````
16:36:23.525 [DCOP-nodeB0] ERROR com.bbn.map.dcop.DCOPService - Got error computing DCOP plan. Skipping this round and will try again next round
org.protelis.lang.interpreter.util.ProtelisRuntimeException: getDcopInfoProvider[] does not exist in class com.bbn.map.dcop.CdiffPlusExecutionContext.
You tried to invoke it with arguments []
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:132) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:300) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:295) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:77) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:300) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.lambda$projectAndEval$2(AbstractAnnotatedTree.java:256) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree$$Lambda$3.accept(AbstractAnnotatedTree.java) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.forEachWithIndex(AbstractAnnotatedTree.java:164) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.projectAndEval(AbstractAnnotatedTree.java:255) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AssignmentOp.evaluate(AssignmentOp.java:45) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All.lambda$evaluate$0(All.java:52) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All$$Lambda$1.accept(All.java) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.forEachWithIndex(AbstractAnnotatedTree.java:164) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All.evaluate(All.java:50) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.vm.impl.SimpleProgramImpl.compute(SimpleProgramImpl.java:76) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.vm.ProtelisVM.runCycle(ProtelisVM.java:45) ~[protelis-interpreter-12.1.0.jar:?]
	at com.bbn.map.dcop.CdiffPlusAlgorithm.computePlan(CdiffPlusAlgorithm.java:70) ~[main/:?]
	at com.bbn.map.dcop.DCOPService.execute(DCOPService.java:121) ~[main/:?]
	at com.bbn.map.AbstractPeriodicService.executeService(AbstractPeriodicService.java:44) ~[main/:?]
	at com.bbn.map.AbstractService.lambda$0(AbstractService.java:89) ~[main/:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
Caused by: java.lang.UnsupportedOperationException: getDcopInfoProvider[] does not exist in class com.bbn.map.dcop.CdiffPlusExecutionContext.
You tried to invoke it with arguments []
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:408) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:424) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.invokeFieldable(ReflectionUtils.java:148) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:110) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	... 23 more
Caused by: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalArgumentException: No accessible method named getDcopInfoProvider callable with [] parameters is available in class com.bbn.map.dcop.CdiffPlusExecutionContext
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2051) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.get(LocalCache.java:3953) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3976) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4960) ~[guava-27.1-jre.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:397) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:424) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.invokeFieldable(ReflectionUtils.java:148) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:110) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	... 23 more
Caused by: java.lang.IllegalArgumentException: No accessible method named getDcopInfoProvider callable with [] parameters is available in class com.bbn.map.dcop.CdiffPlusExecutionContext
	at org.protelis.lang.interpreter.util.ReflectionUtils.loadBestMethod(ReflectionUtils.java:254) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.access$000(ReflectionUtils.java:51) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils$1.load(ReflectionUtils.java:61) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils$1.load(ReflectionUtils.java:56) ~[protelis-interpreter-12.1.0.jar:?]
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.get(LocalCache.java:3953) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3976) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4960) ~[guava-27.1-jre.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:397) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:424) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.invokeFieldable(ReflectionUtils.java:148) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:110) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	... 23 more
16:36:23.525 [DCOP-nodeA0] ERROR com.bbn.map.dcop.DCOPService - Got error computing DCOP plan. Skipping this round and will try again next round
org.protelis.lang.interpreter.util.ProtelisRuntimeException: getDcopInfoProvider[] does not exist in class com.bbn.map.dcop.CdiffPlusExecutionContext.
You tried to invoke it with arguments []
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:132) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:300) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:295) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:77) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.evalInNewStackFrame(AbstractAnnotatedTree.java:300) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.lambda$projectAndEval$2(AbstractAnnotatedTree.java:256) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree$$Lambda$3.accept(AbstractAnnotatedTree.java) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.forEachWithIndex(AbstractAnnotatedTree.java:164) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.projectAndEval(AbstractAnnotatedTree.java:255) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AssignmentOp.evaluate(AssignmentOp.java:45) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All.lambda$evaluate$0(All.java:52) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All$$Lambda$1.accept(All.java) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.forEachWithIndex(AbstractAnnotatedTree.java:164) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.All.evaluate(All.java:50) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.vm.impl.SimpleProgramImpl.compute(SimpleProgramImpl.java:76) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.vm.ProtelisVM.runCycle(ProtelisVM.java:45) ~[protelis-interpreter-12.1.0.jar:?]
	at com.bbn.map.dcop.CdiffPlusAlgorithm.computePlan(CdiffPlusAlgorithm.java:70) ~[main/:?]
	at com.bbn.map.dcop.DCOPService.execute(DCOPService.java:121) ~[main/:?]
	at com.bbn.map.AbstractPeriodicService.executeService(AbstractPeriodicService.java:44) ~[main/:?]
	at com.bbn.map.AbstractService.lambda$0(AbstractService.java:89) ~[main/:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
Caused by: java.lang.UnsupportedOperationException: getDcopInfoProvider[] does not exist in class com.bbn.map.dcop.CdiffPlusExecutionContext.
You tried to invoke it with arguments []
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:408) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:424) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.invokeFieldable(ReflectionUtils.java:148) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:110) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	... 23 more
Caused by: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalArgumentException: No accessible method named getDcopInfoProvider callable with [] parameters is available in class com.bbn.map.dcop.CdiffPlusExecutionContext
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2051) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.get(LocalCache.java:3953) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3976) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4960) ~[guava-27.1-jre.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:397) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:424) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.invokeFieldable(ReflectionUtils.java:148) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:110) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	... 23 more
Caused by: java.lang.IllegalArgumentException: No accessible method named getDcopInfoProvider callable with [] parameters is available in class com.bbn.map.dcop.CdiffPlusExecutionContext
	at org.protelis.lang.interpreter.util.ReflectionUtils.loadBestMethod(ReflectionUtils.java:254) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.access$000(ReflectionUtils.java:51) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils$1.load(ReflectionUtils.java:61) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils$1.load(ReflectionUtils.java:56) ~[protelis-interpreter-12.1.0.jar:?]
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.get(LocalCache.java:3953) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3976) ~[guava-27.1-jre.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4960) ~[guava-27.1-jre.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:397) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:424) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.util.ReflectionUtils.invokeFieldable(ReflectionUtils.java:148) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.DotOperator.evaluate(DotOperator.java:110) ~[protelis-interpreter-12.1.0.jar:?]
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:126) ~[protelis-interpreter-12.1.0.jar:?]
	... 23 more

````

**Comment by DanySK** [comment]
I'm having difficulties reproducing this. Seems to be working in testing environment. Two questions:

1. Is the issue reproducible with recent protelis? Stacktraces report `protelis-interpreter-12.1.0.jar`, version `13.0.3` should be available as we speak
2. Are you sure you are not catching the exception somewhere and "silencing" it? Protelis stacktraces can only get printed if their `printStacktrace` method is called (or the result of `toString` is printed).

If I run `self.getDcopInfoProvider()` I get:
```
org.protelis.lang.interpreter.util.ProtelisRuntimeException: getDcopInfoProvider() does not exist in class org.protelis.test.infrastructure.DummyContext.
You tried to invoke it with arguments[]
	in main script  at line number 1
Fully detailed interpreter trace:
org.protelis.lang.interpreter.util.ProtelisRuntimeException: getDcopInfoProvider() does not exist in class org.protelis.test.infrastructure.DummyContext.
You tried to invoke it with arguments[]
	at: DummyContext701681793.getDcopInfoProvider() at line number 1
java.lang.UnsupportedOperationException: getDcopInfoProvider() does not exist in class org.protelis.test.infrastructure.DummyContext.
You tried to invoke it with arguments[]
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:463)
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:480)
	at org.protelis.lang.interpreter.util.ReflectionUtils.invokeFieldable(ReflectionUtils.java:189)
	at org.protelis.lang.interpreter.impl.Invoke.evaluate(Invoke.java:110)
	at org.protelis.lang.interpreter.impl.AbstractAnnotatedTree.eval(AbstractAnnotatedTree.java:123)
	at org.protelis.vm.impl.SimpleProgramImpl.compute(SimpleProgramImpl.java:65)
	at org.protelis.vm.ProtelisVM.runCycle(ProtelisVM.java:44)
	at org.protelis.test.ProgramTester.runProgram(ProgramTester.java:233)
	at org.protelis.test.ProgramTester.lambda$1(ProgramTester.java:118)
	at org.junit.Assert.assertThrows(Assert.java:1003)
	at org.protelis.test.ProgramTester.runExpectingErrors(ProgramTester.java:114)
	at org.protelis.test.TestStacktrace.testRuntimeErrorOnNonExistingSelfMethod(TestStacktrace.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:305)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:365)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:330)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:78)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:328)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:65)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:292)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:305)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:412)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
Caused by: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalArgumentException: No accessible method named getDcopInfoProvider callable with [] parameters is available in class org.protelis.test.infrastructure.DummyContext
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2051)
	at com.google.common.cache.LocalCache.get(LocalCache.java:3953)
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3976)
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4960)
	at org.protelis.lang.interpreter.util.ReflectionUtils.searchBestMethod(ReflectionUtils.java:449)
	... 37 more
Caused by: java.lang.IllegalArgumentException: No accessible method named getDcopInfoProvider callable with [] parameters is available in class org.protelis.test.infrastructure.DummyContext
	at org.protelis.lang.interpreter.util.ReflectionUtils.loadBestMethod(ReflectionUtils.java:300)
	at org.protelis.lang.interpreter.util.ReflectionUtils.access$0(ReflectionUtils.java:286)
	at org.protelis.lang.interpreter.util.ReflectionUtils$1.load(ReflectionUtils.java:62)
	at org.protelis.lang.interpreter.util.ReflectionUtils$1.load(ReflectionUtils.java:1)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045)
	... 41 more
```

**Comment by DanySK** [comment]
TODO: try to do the same with a named module and make sure the exception prints out the module name

**Comment by DanySK** [comment]
Fixed in [13.1.0](https://github.com/Protelis/Protelis/releases/tag/13.1.0)
